<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointerMotionForm</class>
 <widget class="QWidget" name="PointerMotionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pointer Motion</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="KTitleWidget" name="ktitlewidget">
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="indent_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Minimum:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CustomSlider" name="kcfg_MinSpeed">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Maximum:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CustomSlider" name="kcfg_MaxSpeed">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Acceleration:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="CustomSlider" name="kcfg_AccelFactor">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="KTitleWidget" name="ktitlewidget_2">
     <property name="text">
      <string>Noise cancellation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Vertical hysteresis:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KDoubleNumInput" name="kcfg_VertHysteresis">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="suffix">
         <string>mm</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="sliderEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Horizontal hysteresis:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KDoubleNumInput" name="kcfg_HorizHysteresis">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="suffix">
         <string>mm</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="sliderEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
  <customwidget>
   <class>CustomSlider</class>
   <extends>QSlider</extends>
   <header>customslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
